tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+        default:
+          $ref: '#/responses/Error'
+  /v2/update-tags:
+    post:
+      tags:
+        - tags
+      summary: 'Create or update multiple tags'
+      description: Create or update multiple tags in one request.
+      operationId: CreateOrUpdateTags
+      parameters:
+        - in: body
+          name: tagsAndMerge
+          description: The tags to create or update, and a flag specifying whether keywords and properties should be merged for updates.
+          schema:
+            type: object
+            title: Tag List and Merge Flag
+            properties:
+              tags:
+                description: List of tags
+                type: array
+                items:
+                  $ref: '#/definitions/Tag'
+              merge:
+                description: 'If set, performs a merge of any specified Keywords or Properties.'
+                type: boolean
+          required: true
+      responses:
+        201:
+          description: Created
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: 'Conflict: Type change not supported'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags:
+    post:
+      tags:
+        - tags
+      summary: Create or update a tag
+      description: Create or update single tag. Performs a merge of any specified Keywords or Properties. Changing a tag's type is not supported.
+      operationId: CreateTag
+      parameters:
+        - in: body
+          name: tags
+          description: Tag to create
+          schema:
+            $ref: '#/definitions/Tag'
+          required: true
+      responses:
+        201:
+          description: 'Created: Tag created successfully'
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: 'Conflict: Type change not supported'
+        default:
+          $ref: '#/responses/Error'
+    get:
+      tags:
+        - tags
+      summary: Get multiple tags
+      description: Get a list of tags, filtered based on the provided query parameters.
+      operationId: GetTags
+      parameters:
+        - in: query
+          name: path
+          description: Comma-separated list of tag paths. May include glob-style wildcards
+          type: string
+          required: false
+        - in: query
+          name: keywords
+          description: Comma-separated list of case-sensitive keywords.
+          type: string
+          required: false
+        - in: query
+          name: properties
+          description: Comma-separated list of name=value strings, e.g. key1=value,key2=value.
+          type: string
+          required: false
+        - in: query
+          name: skip
+          description: Used for pagination. Specifies the number of entries to skip in the response list. Typically used in combination with the take parameter.
```

Comment:
Number of entries to skip in the response list. Use with the 'take' parameter to [what do they use it for? pagination?]. 

2 questions here: 
1. Does "used for pagination" mean it helps jump around in a list of responses? I couldn't find an industry-specific definition for pagination.
2. Is there a way to clarify that take is a parameter? I used single quotes above but I'm not sure if that's misleading.

Reply:
They use it for pagination, yes. I removed the first sentence and reworded it like this: Number of entries to skip in the response list. Typically used in combination with the take parameter to implement pagination.

1. Yes. If your query returned too many tags to include in one response, you use skip and take to get the rest of your data. I think skip and take are pretty intuitive for clients of REST APIs, so I'm not too worried about it.
2. Take is specified under the "parameters" section, so it appears in Swagger UI's "parameters" section. I think that should be sufficient.